gensim/similarities/docsim.py

```diff
@@ -116,8 +131,20 @@ def get_index(self):
     def get_document_id(self, pos):
         """Return index vector at position `pos`.
 
+        Parameters
+        ----------
+        pos : int
+            Vector position.
+
+        Return
+        ------
+        {:class: `~scipy.sparse.csr_matrix`, :class: `~numpy.array`}
```

Comment:
`numpy.array` -> `numpy.ndarray` here and everywhere. Also, in this case, link shouldn't be rendered -> don't use `~` for numpy/scipy